modules/utils/filters.js

```diff
@@ -1,7 +1,14 @@
+import sanitizeHtmlFunction from 'sanitize-html-react';
```

Comment:
I'll move this to `core` module — modules are feature-based so `utils` as its own module feels little wonky.

We've been using the `core` module as "catch-all", tho I agree we could have some better place for common things.